core/src/components/datetime/test/minmax/datetime.e2e.ts

```diff
@@ -301,21 +301,20 @@ configs({ directions: ['ltr'], modes: ['ios'] }).forEach(({ title, config }) =>
           <ion-datetime
             min="2022-01-15"
             value="2022-02-01"
-            presentation="date"
+            presentation="month-year"
```

Comment:
Was this change intentional?

Reply:
Yup! So this test was checking the month/year picker to make sure it updates the value prop correctly. Previously it was using the grid style layout, clicking the month/year toggle button, then changing the values. This simplifies it by just using the month-year layout to begin with.

Reply:
Ahh, clever 👍